roles/download/defaults/main.yml

```diff
@@ -117,7 +117,7 @@ kibana_version: "v4.6.1"
 kibana_image_repo: "gcr.io/google_containers/kibana"
 kibana_image_tag: "{{ kibana_version }}"
 
-helm_version: "v2.2.2"
+helm_version: "v2.7.0"
```

Comment:
will not work as here https://hub.docker.com/r/lachlanevenson/k8s-helm/tags/
no image *v2.7.0* (2.7.0 is not v2.7.0)
BTW: it is verified but v2.6.2 works